### PR TITLE
create mohammad_project.md

### PR DESCRIPTION
**Pull request by mtorabi59**
Adding my project description

### CODE REVIEW

**Comment by mtorabi59** [comment]
@surchs Would be grateful if you accept the pull request!

File: _projects/mohammad_project.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Improved Assessment of Brain dynamic Functional Connectivity
+status: active
+layout: project
+people:
+  - Mohammad
```
Comment (surchs) [inline _projects/mohammad_project.md:6]: This should match your [name in the people.yml file](https://github.com/neurodatascience/neurodatascience.github.io/blob/gh-pages/_data/people.yml#L120). I think it does!

File: _projects/mohammad_project.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+Functional Connectivity (FC) has emerged as an important measure for understanding brain function as well as a biomarker with significant potential. 
+FC assessment was initially performed under the stationarity assumption. However, growing evidence supports that the dynamical variations of FC (dynamic FC – dFC) has a critical role related to brain function (Savva et al., 2019). FC has been commonly assessed using functional Magnetic Resonance Imaging (fMRI) data. Despite its excellent spatial resolution, fMRI has a low temporal resolution, making fMRI controversial for being used for a “high-temporal-resolution-requiring” measurement like dFC. Therefore, one of the main goals of this project is to enhance the temporal resolution of fMRI data. Built upon the enhanced fMRI data, we will then be able to carry out a high-resolution dFC assessment which will promote us a deeper understanding of dFC sources and more importantly, brain functional organization.
```
Comment (surchs) [inline _projects/mohammad_project.md]: I like that you are providing references. In that case I would add either a link or DOI at the end of the document for an interested reader

File: _projects/mohammad_project.md (hunk 1 above)
Comment (mtorabi59) [reply _projects/mohammad_project.md:6]: I see. Yes it does!

File: _projects/mohammad_project.md (hunk 2 above)
Comment (mtorabi59) [reply _projects/mohammad_project.md]: Good idea!

Comment (surchs) [reply _projects/mohammad_project.md]: I think a better word for "promote" here would be "provide"
```suggestion
FC assessment was initially performed under the stationarity assumption. However, growing evidence supports that the dynamical variations of FC (dynamic FC – dFC) has a critical role related to brain function (Savva et al., 2019). FC has been commonly assessed using functional Magnetic Resonance Imaging (fMRI) data. Despite its excellent spatial resolution, fMRI has a low temporal resolution, making fMRI controversial for being used for a “high-temporal-resolution-requiring” measurement like dFC. Therefore, one of the main goals of this project is to enhance the temporal resolution of fMRI data. Built upon the enhanced fMRI data, we will then be able to carry out a high-resolution dFC assessment which will provide with us a deeper understanding of dFC sources and more importantly, brain functional organization.
```

**Review by surchs — APPROVED** [review]
looks good to me! Congrats @mtorabi59

**Comment by mtorabi59** [comment]
@surchs  okay I made the changes! Should pull request again?

**Comment by surchs** [comment]
@mtorabi59, when you have made all the changes you wanted to make, you can go ahead and merge this pull request!

**Pull request merged**